src/View/Helper/FormHelper.php

```diff
@@ -367,11 +367,16 @@ public function create($model = null, array $options = [])
         }
         unset($options['templates']);
 
-        $url = $this->_formUrl($context, $options);
-        $action = $this->Url->build($url);
-        unset($options['url'], $options['action'], $options['idPrefix']);
+        if ($options['url'] === false) {
+            $action = null;
+            $this->_lastAction = '';
```

Comment:
Forms with no action attribute post to the current URL. Blanking this property will result in SecurityComponent hashes not having the URL and subsequently failing validation.